### ISSUE TITLE
Claire Cui

### ISSUE THREAD

**Issue by claire43** [cucapra/undergrad-research#18]
## Personal Details

**Name**: Claire Cui

**Major**: Computer Science

**Year in Cornell**: Junior

**Relevant classes**: 
Courses relevant to PL and Architecture: CS 3110(A+, TA for two semesters), 3410(A+)
Others: CS 1110/2110/2800/4154/4820/5152/4620/4780
Math 3110/4310/4710, ORIE 3300

## Research

**When do you want to do research?** In the semester, in particular Fall 2019

**What is exciting to you about research?**
- I've always wished to go beyond the classroom knowledge and try solving new problems based on what I've mastered in courses.
- I enjoy studying programming languages. CS 3110 has so far been my favorite class at Cornell. Unfortunately 4110 is not offered this fall, but this doesn't stop me from seeking to learn more about it through a potential research project!
- I worked in the industry this summer, and I would love to get a taste of research in order to make more informed decisions for the future. 

**What kind of research do you want to do?** 
As a person who enjoys math and logic, I used to think that I would only be passionate about types of work that are more high-level, theoretical and abstract. It was not until finishing CS 3410 that I realized I was actually attracted to the full spectrum, from functors and monads to gates and circuits. While I had pretty good ideas of how things work on both ends, I would love to explore more work that lies somewhere in between, at the interface of hardware/software. 

Having been a software engineering intern in the summer and having taken/am taking software engineering classes (4154 and 5152) at Cornell, I have solid programming skills and I love implementing stuff.

That being said I'm also very passionate about high level programming abstractions. 

## Optional Stuff

**Is there a specific grad student or a project you're interested in working on?**
I spoke to Alexa at the research night and find her current work interesting. Both building cost models and rewriting implementation in C++ sound like great projects that I would love to work on. 

**Anything else you want to tell us about yourself?**
I am available to meet Fridays after 2:30pm, MW after 4:30pm, TR after 4:15pm and TR 1-2:30pm. 

**Attach a CV if you like.**
https://www.linkedin.com/in/claire-yuqing-cui/

**Comment by avanhatt** [comment]
Hey Claire, thanks for reaching out! I sent you an email with more details.

**Comment by claire43** [comment]
Hi Alexa, I was wondering if there is anything I could help with or if there are any other projects going on in the lab for this summer. I will be able to work full-time remotely. Thank you!

**Comment by avanhatt** [comment]
Hi Claire, we may have an opportunity for remote research this summer (it might, however, be contingent on the remote work being done somewhere in the US as opposed to abroad). If you are still interested, please email me at avh@cs.cornell.edu and we can set up a meeting time to discuss further.

**Comment by rachitnigam** [comment]
Not sure if @claire43 and @avanhatt already had a chat. @claire43 if you're still looking for a research position, please open a new issue. This issue doesn't use the new template.
